### PR TITLE
Add docs about "Ignore transactions"

### PR DESCRIPTION
**Pull request by vermaxik**
Hi,

It would be nice to find documentation about "Ignoring transactions" in README and Hexdocs.

I hope this MR will be useful for other people as well.

BR, Maxim

### CODE REVIEW

**Comment by binaryseed** [comment]
Looks like your git config isn't configured correctly, preventing the CLA tool from knowing what your github user is..

**Comment by vermaxik** [comment]
> Looks like your git config isn't configured correctly, preventing the CLA tool from knowing what your github user is..

Thank you, fixed. Now specs if failed for one build, but for me it looks fine. Is possible to retry?

File: README.md [hunk 1]
```diff
@@ -95,3 +95,13 @@ end
 alias NewRelic.Instrumented.HTTPoison
 HTTPoison.get("http://www.example.com")
 ```
+
+#### Ignore transactions
```
Comment (binaryseed) [inline README.md]: Would you mind removing the changes to the `README` file please? I'd like to keep the contents very high level

Comment (vermaxik) [reply README.md]: Yeah, sure. The section was removed.

**Comment by binaryseed** [comment]
Thanks!

**Pull request merged**